search-endpoint.md

```diff
@@ -223,12 +223,12 @@ The returned JSON response will be like:
               "dc.publisher" : "My <em>very cool</em> publisher"
             },
             "_links" : {
-              "dspaceObject" : {
+              "rObject" : {
```

Comment:
Silly question, but what does `rObject` refer to?  I understand that some results now may not be true "DSpaceObject" classes (in the Java sense), but I don't understand what `rObject` means.    Is it maybe a "resultObject" -- if so, I wonder if we should simply call this `resultObject`?  Or, should we consider just keeping this named `dspaceObject`, even though we all realize it's not necessarily returning a Java `DSpaceObject` class?

As a sidenote, this simple name change likely will need to be analyzed on the Angular side.  I'd expect it to break Angular UI parsing of search results (but hopefully it'd be easy to fix).

Reply:
yes, rObject was a shortcut for resultObject. The change in the json is already managed in the MyDSpace Angular PR that @atarix83 will open as soon as we agree about the indexing approach. If preferred I can change it to resultObject, this will require (minor) changes in the ITs and on the coming @atarix83 PR

Reply:
@tdonohue renamed in resultObject as agreed via slack